roles/common/templates/auto-upgrades.j2

```diff
@@ -0,0 +1,4 @@
+APT::Periodic::Update-Package-Lists "1";
+APT::Periodic::Unattended-Upgrade "1";
+Unattended-Upgrade::Automatic-Reboot "true";
+Unattended-Upgrade::Automatic-Reboot-Time "02:00";
```

Comment:
This isn't great for reliability, as it means that all servers will reboot at the same time! 

However, since there's only one server, and we don't have hard reliability requirements, I'm going to let this pass.